argo-cd-apps/overlays/konflux-public-production/kustomization.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/host
+  - ../../base/member
+  - ../../base/all-clusters
+  - ../../base/tekton-ci
+  - ../../base/tenants-config
+  - ../../base/cluster-secret-store-rh
+  - ../../base/rh-managed-workspaces-config
+
+namespace: argocd
```

Comment:
is this the correct namespace?

Reply:
i see it's a duplicate. can you please remove it?

Reply:
accidentally added two namespace, fixed now.